DEPLOYMENT_GUIDE.md

```
@@ -0,0 +1,258 @@
+# Environnement
+NODE_ENV=production
+
+# Synchronisation automatique (optionnel, activé par défaut en production)
+ENABLE_AUTO_SYNC=true
+```
+
+## 🚀 Méthode 1 : Déploiement automatique via GitHub (Recommandé)
+
+### Étape 1 : Merger la Pull Request
+
+1. Accédez à la PR : https://github.com/doriansarry47-creator/planning/pull/45
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)